base/array.jl

```diff
@@ -1443,6 +1443,10 @@ If `first` is true, then the reserved space is from the start of the collection,
 collections. Supplying this keyword may result in an error if the collection is nor ordered
 or if `pushfirst!` is not supported for this collection.
 
+If `shrink` is `true` (the default), the reserved space may be reduced if it is
```

Comment:
I'll try to rephrase the whole paragraph:

By passing `shrink=false` one may suggest that the reserved space is preserved
even if it can be reduced, i.e., the the collection is not shrunk.

Reply:
I find it not super logical to describe what `shrink=false` does first, as it's a negative proposition, without mentioning first what the positive action of shrinking (which is the default) does. This is even more surprising if the first sentence is reverted to "at least `n` elements", which doesn't imply at all that the reserved space may be shrunk.

Reply:
I think it probably makes sense to document this as
```
If `shrink=true` (the default), the collection's capacity may be reduced if its current capacity is greater than `n`
```